src/IO/lbsLoadReaderVT.py

```diff
@@ -162,102 +162,102 @@ def read_iteration(self, n_p: int, phase_id: int) -> list:
             proc_objects_set.update(proc.get_objects())
         proc_objects_dict = {obj.get_id(): obj for obj in proc_objects_set}
 
-        # iterating over processors
+        # iterating over ranks
         for proc_num, proc in enumerate(procs):
-            # iteration over objects in processor
+            # iteration over objects in rank
             for proc_obj in proc.get_objects():
                 obj_id = proc_obj.get_id()
                 # checking if there is any communication for the object
                 obj_communication = communication.get(proc_num, None).get(obj_id, None)
                 if obj_communication is not None:
-                    send = {proc_objects_dict.get(snd.get('to'), None): snd.get('bytes') for snd in
-                            obj_communication.get('send') if proc_objects_dict.get(snd.get('to'), None) is not None}
-                    received = {proc_objects_dict.get(snd.get('from'), None): snd.get('bytes') for snd in
-                                obj_communication.get('received') if
-                                proc_objects_dict.get(snd.get('from'), None) is not None}
+                    send = {proc_objects_dict.get(snd.get("to"), None): snd.get("bytes") for snd in
```

Comment:
What's the point of changing quotation marks?

Reply:
I started like this in LBAF so that's just for consistency of style throughout the code